README.md

```diff
@@ -1,24 +1,126 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+# 📖 Blog App <a name="about-project"></a>
 
-* System dependencies
+**Budget-app** Budget app built with Ruby on Rails. You can create categories and have a list of transactions associated with a category, you can add new transactions and see the money you spent in each category and transaction.
 
-* Configuration
+## 🛠 Built With <a name="built-with"></a>
 
-* Database creation
+Ruby on Rails, Visual Studia Code, WSL
 
-* Database initialization
+### Tech Stack <a name="tech-stack"></a>
 
-* How to run the test suite
+- **Ruby on Rails**
 
-* Services (job queues, cache servers, search engines, etc.)
+- **WSL**
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+> - List Categories
+> - Add a new Category
+> - List Transactions
+> - Add a new Transaction
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+LIVE DEMO
+## 🚀 Live Demo <a name="live-demo"></a> 
+
+
+
+- [Live Demo Link](https://budget-app-1-l7xh.onrender.com)
```

Comment:
👍 Excellent progress on implementing the signup functionality! However, there seems to be an issue after creating a new category, resulting in a server-side error (`We're sorry, but something went wrong`). While I may not have direct access to your server logs, I recommend checking them for more details on the error.

Possible steps to troubleshoot:

- 📑 **Check Server Logs:** Examine the server logs to identify the specific error and traceback information. This will provide valuable insights into what went wrong.
- 🧐 **Debug the Code:** Inspect the code related to category creation and ensure that it aligns with the project requirements. Pay attention to any exceptions or errors that might occur during category creation.
- 🚀 **Deployment:** If you've made changes, ensure that you've redeployed your application to apply the latest code updates.
![image](https://github.com/vikas924/budget-app/assets/55840999/dfb7a586-388b-4e95-81f8-fc2813cc473d)


Once you've identified the root cause of the server-side error, you'll be better equipped to implement a solution. Keep up the good work! 👏💻